examples/nlp/language_modeling/megatron_gpt_eval.py

```diff
@@ -202,9 +203,20 @@ def main(cfg) -> None:
         == cfg.tensor_model_parallel_size * cfg.pipeline_model_parallel_size
     ), "devices * num_nodes should equal tensor_model_parallel_size * pipeline_model_parallel_size"
 
-    # Load prompt tuned model, model_file must be provided in config
-    if cfg.get('virtual_prompt_model', False):
-        model = MegatronGPTPromptLearningModel.restore_from(cfg.model_file, trainer=trainer)
+    # Load prompt tuned model, virtual_prompt_model_file must be provided in config
+    if cfg.get('virtual_prompt_model_file', None) is not None:
+
+        # Update frozen GPT model path in case it has changed
```

Comment:
Oh the double extraction is gonna be really costly for LLM. I think unavoidable though.

Reply:
Double extraction cost should actually be really small because this .nemo file only contains virtual prompt weights, not gpt model weights.

Reply:
Perfect.